corehq/apps/data_interfaces/interfaces.py

```diff
@@ -117,11 +120,16 @@ def bulk_response(self):
                 _("An owner_id needs to be specified to bulk reassign cases")
             )
 
+        # If we use self.es_results we're limited to the pagination set on the
+        # UI by the user
+        es_results = self._es_query\
+            .size(self.total_records)\
```

Comment:
Just to make sure I understand this correctly: `self._es_query` calls `self._build_query()`, which uses the `.size(self.pagination.count)`. Does this here call to `.size(self.total_records)` override the previously specified size?

Reply:
@SmittieC Yes, it overrides the size.